### PR TITLE
Create authentication

### LINKED ISSUE #3: Create authentication (login via email) ~ 6 hrs
NextAuth authentications using credentials.

### PR DESCRIPTION
**Pull request by Tek58**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+    "plugins": ["@typescript-eslint"],
     "extends": [
-      // "next/babel",
       "next",
       "next/core-web-vitals",
       "plugin:@typescript-eslint/recommended",
       "prettier"
     ],
     "rules": {
-      "@typescript-eslint/no-unused-vars": "error",
```
Comment (danielwollen) [inline .eslintrc:11]: Why did you remove that check?

File: package.json [hunk 2]
```diff
@@ -9,25 +9,30 @@
...
+    "react": "16.14.0",
+    "react-dom": "16.14.0"
   },
   "devDependencies": {
+    "@types/bcrypt": "^5.0.0",
+    "@types/next-auth": "3.1.14",
+    "@types/node": "14.14.6",
     "@tailwindcss/jit": "^0.1.18",
-    "@types/react": "17.0.20",
+    "@types/react": "16.9.56",
```
Comment (danielwollen) [inline package.json:24]: Why did you downgrade to react 16?

**Review by danieloyo — COMMENTED** [review]
-

File: package.json (hunk 2 above)
Comment (Tek58) [inline package.json:24]: There was a problem with the tailwindcss, postcss and autoprefixer if the versions were to be the latest so as node. since I had faced the exact problem before on the previous task, I used exact versions of the previous ones that worked

File: .eslintrc (hunk 1 above)
Comment (Tek58) [inline .eslintrc:11]: when working on nextauth, there was one callback function for redirection that had parameters that were not used, and it kept showing me errors, that's y I removed it, coz i wasn't using them anywhere, I have removed the code it was for testing
